src/main/java/de/fuberlin/chaostesting/ExecuteTestAction.java

```diff
@@ -110,4 +110,15 @@ public Resolution executeTest() {
 		
 		return new ForwardResolution("/executeTest.jsp");
 	}
+	
+	public String Validation(String xml){
```

Comment:
This method can be private. Use verbs for method names. Start method names with lowercase letters.